db/seeds.rb

```diff
@@ -12,28 +12,3 @@
 user.password = 'password'
 user.save
 Ability.new(user)
```

Comment:
I would have if a) I knew how to set the password with fixtures b) didn't use that user all the time during normal development